.ci/scripts/go-plus/tgc-tester-integration/test_tgc_integration.sh

```diff
@@ -38,7 +38,7 @@ post_body=$( jq -n \
 
 curl \
   -X POST \
-  -u "$github_username:$GITHUB_TOKEN" \
+  -u "$github_username:$GITHUB_TOKEN_CLASSIC" \
```

Comment:
This needs to match what's being passed in.

```suggestion
  -u "$github_username:$GITHUB_TOKEN_MAGIC_MODULES" \
```